documentation/modules/upgrading/con-upgrade-custom-resources.adoc

```diff
@@ -0,0 +1,104 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='con-upgrade-custom-resources-{context}']
+
+= API versioning
+
+[role="_abstract"]
+Custom resources are created as instances of APIs added by CRDs to extend Kubernetes resources.
+CRDs are installed in a Kubernetes cluster to define the schemas used to instantiate and manage the custom resources.
+
+For release {ProductVersion}, Strimzi CRDs must specify `{ApiVersion}` as the API version and `{KafkaApiVersion}` as the API path for custom resources.
+Strimzi Operators access custom resources using the API path.
+
+The introduction of `{ApiVersion}` updates the schemas of the custom resources.
+Older API versions are deprecated.
+
+The `v1alpha1` API version is deprecated for the following Strimzi resources:
+
+* `Kafka`
+* `KafkaConnect`
+* `KafkaConnectS2I`
+* `KafkaConnector`
+* `KafkaMirrorMaker`
+* `KafkaMirrorMaker2`
+* `KafkaTopic`
+* `KafkaUser`
+* `KafkaBridge`
+* `KafkaRebalance`
+
+The `v1beta1` API version is deprecated for the following Strimzi resources:
+
+* `Kafka`
+* `KafkaConnect`
+* `KafkaConnectS2I`
+* `KafkaMirrorMaker`
+* `KafkaTopic`
+* `KafkaUser`
+
+IMPORTANT: The `v1alpha1` and `v1beta1` versions will be removed in the next minor release.
+
+API versioning policy indicates the relative stability of the version.
+Alpha version APIs are the least mature, beta versions more mature, and stable versions represent a durable release.
+
+Example API versions:
+
+* Alpha version: `v1alpha1`
+* Beta version: `v1beta1`, `v1beta2`
+* Stable version: `v1`
+
+After a new custom resource type is added to a cluster by installing a CRD, you can create instances of the resource based on its specification.
+Strimzi CRDs strictly adhere to a structural schema specification.
```

Comment:
Actually it's only v1beta2 which is structural.

Reply:
Changed to: _Custom resources using `{ApiVersion}` strictly adhere to a structural schema specification._